crates/bevy_ecs/src/change_detection.rs

```diff
@@ -95,21 +95,21 @@ macro_rules! change_detection_impl {
             #[inline]
             fn is_added(&self) -> bool {
                 self.ticks
-                    .component_ticks
-                    .is_added(self.ticks.last_change_tick, self.ticks.change_tick)
+                    .added
+                    .is_changed(self.ticks.last_change_tick, self.ticks.change_tick)
```

Comment:
The `is_changed` is pretty confusing when used in this context. Can we find a more neutral name?

Reply:
In one of the earlier iterations, I renamed this to `is_older_than`. Would that be better?

Reply:
Yep, I like that!

Reply:
Done.